projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -240,6 +241,24 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         return 0;
     }
 
+    /**
+     * Gets/Sets the data clone strategy of the grid when in edit mode.
+     *
+     * @example
+     * ```html
+     *  <igx-grid #grid [data]="localData" [dataCloneStrategy]="customCloneStrategy"></igx-grid>
+     * ```
+     */
+    @Input()
+    public get dataCloneStrategy(): IDataCloneStrategy {
+        return this._dataCloneStrategy;
+    }
+
+    public set dataCloneStrategy(strategy: IDataCloneStrategy) {
+        this._dataCloneStrategy = strategy;
```

Comment:
check if `strategy` is provided before set the internal field. Otherwise we may end with no strategy.